src/coreclr/jit/emitxarch.cpp

```diff
@@ -14185,8 +14311,7 @@ size_t emitter::emitOutputInstr(insGroup* ig, instrDesc* id, BYTE** dp)
             break;
 
         case IF_AWR_RRD_CNS:
-            assert(ins == INS_vextracti128 || ins == INS_vextractf128);
-            assert(UseVEXEncoding());
```

Comment:
We don't need VEX encoding assert anymore?

Reply:
Right, this applies to 128-bit instructions as well now, such as `Sse41.Extract`